### DISCUSSION TITLE
variant-specific vaccinations-by-manufacturer

### DISCUSSION THREAD

**Discussion by willgilks** [owid/covid-19-data#2732] [General]
Hi, do the vaccinations-by-manufacturer data combine vaccines for different variants within manufacturer. e.g. Moderna BA.1, Moderna BA.4/5, Pfizer BA.1 etc? I suspect the answer is yes. Is there a plan to divide vaccinations by target variant in the future?

**Comment by lucasrodes** [comment]
Hi @willgilks,
We currently aggregate all variants of a manufacturer into one single one, as you correctly guessed.

We currently don't plan to break down the vaccinations by variant, sorry.
